### ISSUE TITLE
WebAPI fails to compile with missing zip4j dependency

### ISSUE THREAD

**Issue by JamesSWiggins** [OHDSI/WebAPI#1315]
v2.7.2 and v2.7.3 give the below error when compiling.  This is a recent thing, as v2.7.2 has compiled well for me for a long time:

[INFO] 4 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.869 s
[INFO] Finished at: 2019-08-28T17:04:57Z
[INFO] Final Memory: 47M/529M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5.1:compile (default-compile) on project WebAPI: Compilation failure: Compilation failure:
[ERROR] /tmp/WebAPI/src/main/java/org/ohdsi/webapi/executionengine/service/AnalysisResultFileContentSensitiveInfoServiceImpl.java:[4,29] error: package net.lingala.zip4j.core does not exist
[ERROR] /tmp/WebAPI/src/main/java/org/ohdsi/webapi/executionengine/service/AnalysisResultFileContentSensitiveInfoServiceImpl.java:[158,12] error: cannot find symbol
[ERROR]   symbol:   class ZipFile
[ERROR]   location: class AnalysisResultFileContentSensitiveInfoServiceImpl
[ERROR] /tmp/WebAPI/src/main/java/org/ohdsi/webapi/executionengine/service/AnalysisResultFileContentSensitiveInfoServiceImpl.java:[158,34] error: cannot find symbol
[ERROR]   symbol:   class ZipFile
[ERROR]   location: class AnalysisResultFileContentSensitiveInfoServiceImpl
[ERROR] /tmp/WebAPI/src/main/java/org/ohdsi/webapi/executionengine/service/ScriptExecutionServiceImpl.java:[148,31] error: Alternatives in a multi-catch statement cannot be related by subclassing
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

However, when I add the below lines to pom.xml, it compiles just fine:
```
<dependency>
    <groupId>net.lingala.zip4j</groupId>
    <artifactId>zip4j</artifactId>
    <version>1.3.3</version>
</dependency>
```

**Comment by pavgra** [comment]
https://github.com/OHDSI/WebAPI/compare/v2.7.3...isssue-1315

**Comment by anthonysena** [comment]
@JamesSWiggins - I'm going to close this out as it appears to be resolved via #1314. Let me know if you have any concerns. Thanks!
